config/settings-production.yml

```diff
@@ -52,12 +52,14 @@ buffy:
     add_remove_checklist:
```

Comment:
hey @xuanxu 👋 no rush on this but i've managed to break things again. 

if you look through the heroku logs you will see 

```
2024-03-14 22:43:10.586 [buffy_pyosbot] router connect= POST buffy-pyosbot-f84ad506ee51.herokuapp.com /dispatch 140.82.115.150 at=error code=H10 desc="App crashed" method=POST path="/dispatch" host=buffy-pyosbot-f84ad506ee51.herokuapp.com request_id=28cf8a4b-e6c5-473e-8528-e9a9dd6e99eb fwd="140.82.115.150" dyno= connect= service= status=503 bytes= protocol=https1 nested result


2024-03-14 22:44:15.826 [buffy_pyosbot] web.1 [2] ! Unable to load application: Psych::SyntaxError: (<unknown>): mapping values are not allowed in this context at line 54 column 15
2024-03-14 22:44:15.826 [buffy_pyosbot] web.1 /app/vendor/ruby-3.3.0/lib/ruby/3.3.0/psych/parser.rb:62:in `_native_parse': (<unknown>): mapping values are not allowed in this context at line 54 column 15 (Psych::SyntaxError)
2024-03-14 22:44:15.826 [buffy_pyosbot] web.1 	from /app/vendor/bundle/ruby/3.3.0/gems/bundler-2.5.6/lib/bundler/friendly_errors.rb:117:in `with_friendly_errors'
2024-03-14 22:53:18.096 [buffy_pyosbot] web.1 [2] ! Unable to load application: Psych::SyntaxError: (<unknown>): mapping values are not allowed in this context at line 54 column 15
2024-03-14 22:53:18.097 [buffy_pyosbot] web.1 /app/vendor/ruby-3.3.0/lib/ruby/3.3.0/psych/parser.rb:62:in `_native_parse': (<unknown>): mapping values are not allowed in this context at line 54 column 15 (Psych::SyntaxError)
2024-03-14 22:53:18.097 [buffy_pyosbot] web.1 	from /app/vendor/bundle/ruby/3.3.0/gems/bundler-2.5.6/lib/bundler/friendly_errors.rb:117:in `with_friendly_errors'
2024-03-14 22:54:57.508 [buffy_pyosbot] router connect= POST buffy-pyosbot-f84ad506ee51.herokuapp.com /dispatch 140.82.115.92 at=error code=H10 desc="App crashed" method=POST path="/dispatch" host=buffy-pyosbot-f84ad506ee51.herokuapp.com request_id=8659c4f7-4d65-4468-b3fb-a1d33e809772 fwd="140.82.115.92" dyno= connect= service= status=503 bytes= protocol=https1 nested result
2024-03-14 22:55:27.916 [buffy_pyosbot] web.1 [2] ! Unable to load application: Psych::SyntaxError: (<unknown>): mapping values are not allowed in this context at line 54 column 15
2024-03-14 22:55:27.917 [buffy_pyosbot] web.1 /app/vendor/ruby-3.3.0/lib/ruby/3.3.0/psych/parser.rb:62:in `_native_parse': (<unknown>): mapping values are not allowed in this context at line 54 column 15 (Psych::SyntaxError)
2024-03-14 22:55:27.917 [buffy_pyosbot] web.1 	from /app/vendor/bundle/ruby/3.3.0/gems/bundler-2.5.6/lib/bundler/friendly_errors.rb:117:in `with_friendly_errors'
```

this error aligns with this add_Remove_checklist element that i added.

similar to the labels i tried to turn it into an array of items for each action i want to perform. i have editor response and package accepted. 

i've played with

* the spacing hierarchy 
* making sure the commands were specified.  command: editor response
* Making sure i have the right spelling of each template item (we do have a mix of underscores and dashes now which i can fix)

but it's still erroring at line 54

is it possible that it's erroring because the package-name key doesn't exist in the template yet? or does something else jump out to you that i'm missing? i thought i could troubleshoot this one but i'm stuck again missing a syntax issue somewhere.

Reply:
When creating the array, you were using a string as the key for each instance, but a key must end with `:`and spaces are not allowed, [here is the correction](https://github.com/pyOpenSci/buffy/commit/eb52bd1c63b4eb9c744743b8bd27662add9fc544). 
I also updated the name of the template to be consistent with using undescores instead of dashes for word separation.